lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -10,25 +10,31 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <cassert>
```

Comment:
Include order changed and it seems to be weird. I doubt `cassert` is necessary (likely we'd get it transitively), but if needed – it should go down.

Reply:
Looks like it was a mistake. Removed.